### PR TITLE
User can be added to an organization

### PR DESCRIPTION
**Pull request by Yagazie-davidson**
User can now be added to an organization from the page navigated to after creating an organization and from my api page.
# TODO
display a better information about the user added like name or email. Right now it shows only role.

### CODE REVIEW

File: src/components/OrganizationInvite.jsx [hunk 1]
```diff
@@ -1,25 +1,49 @@
...
+    <form onSubmit={addUser}>
         <Stack direction="row">
             <Stack>
-                <InputField type= 'text' label= 'Search by Username or Email' placeholder='name@example.com' />
+                <InputField type= 'text' label= 'Search by Username or Email' vlaue={email} onChange={(e) => setEmail(e.target.value)} placeholder='name@example.com' />
             </Stack>
             <Stack direction='row' mx={4} width={155}>
                 <Select style={{ width:'100%', height:'43px', color:"black"}} fullWidth onChange={(e) => setRole(e.target.value)} defaultValue="role">
-                    <MenuItem value='role' onClick={handleClick}>Role</MenuItem>
+                    <MenuItem value='role'>Role</MenuItem>
```
Comment (taiwoAkindele) [inline src/components/OrganizationInvite.jsx:46]: set this to disabled. User should not be able to select it

File: src/pages/CreateOrg.jsx [hunk 2]
```diff
@@ -23,36 +23,25 @@ const useStyles = makeStyles({
 })
 
 const CreateOrg = () => {
-  // const [value, setValue] = useState(' ')
   const classes = useStyles()
   const navigate = useNavigate()
   const { clearError, error, loading, organizationUser } = useOrganizationService()
 
   const [name, setname] = useState("")
   const [number_of_seats, setNumber_of_seats] = useState("")
```
Comment (taiwoAkindele) [inline src/pages/CreateOrg.jsx:31]: Number of seats and employees should not go to negative.

**Review by taiwoAkindele — CHANGES_REQUESTED** [review]
Where do I view the added team mates?

**Comment by Yagazie-davidson** [comment]
>
the page has to be refreshed

**Review by taiwoAkindele — APPROVED** [review]

**Review by Temitopefa — APPROVED** [review]

**Review by PhilemonBrain — APPROVED** [review]

**Pull request merged**